docs/markdown/glossary.md

```diff
@@ -0,0 +1,260 @@
+
+## Destination Amount  
+
+The amount to be received by the receiver through [ILP](#interledger-protocol-ilp).                                                                      
+
+## Distributed Ledger
+
+A ledger that is operated by a group of entities and runs on multiple servers. This term is used somewhat interchangeably with "Blockchain" or "Decentralized Ledger".                                                                        
+
+## Exchange Rate
+
+The price of one ledger's asset in terms of another ledger's asset. Routers may generate revenue from the difference in value between incoming and outgoing transfers. The exchange rate between a sender and a receiver is determined by the product of exchange rates at each hop.                                                                         
+
+## Fulfillment 
+
+Also known as the "preimage" or "hash preimage", a fulfillment is  a 32-byte value used to trigger the execution of a transfer. In most Interledger payments, the fulfillment is known only to the receiver (or in the case of the STREAM protocol it is known to the sender and the receiver).                                                                        
+
+## Hop 
+
+The movement of an ILP packet from one router to another as it travels through the Interledger network to reach its destination.                                                                         
+
+## IL-RFC  
+
+Interledger request for comments. A document describing a part of the protocol stack, or a topic related to it, such as this glossary. The official list of IL-RFCs is maintained in gh:interledger/rfcs.                                                                                                                               
+
+## Interledger (in "Let's use Interledger for that!")  
+
+The Interledger protocol stack.                                                                         
+
+## interledger (in "An interledger network")
+
+A network of two or more ledgers that use the Interledger protocol stack. 
+
+See also: [Interledger (The)](#interledger-in-the-interledger) 
+
+The adjective 'interledger' is also sometimes used to refer to transactions that cross multiple ledgers, even if ILP is not used (similar to 'international' meaning 'across multiple nations').                                                                         
+
+## Interledger (in "The Interledger")  
+
+The public network of ledgers connected via the Interledger protocol stack.                                                                         
+
+## Interledger Address 
+
+An Interledger address provides a way to route ILP packets to their intended destination through a series of hops, including any number of ILP routers. 
+
+An Interledger address is a dot-separated string that contains prefixes to group ledgers. An example address might look like `g.us.acmebank.acmecorp.sales.199` or `g.crypto.bitcoin.1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2`. 
```

Comment:
this is no longer really the case in Interledger. Because the network is now built with connectors and bilateral accounts rather than ledgers and ledger accounts, ILP addresses are now built up of segments which represent connectors. Adding more segments on the end typically means you're referring to a connector's peer with relationship child

Reply:
https://github.com/interledger/interledger.github.io/pull/123/files#diff-0f4b6322d1459dbe5c7f5eb93d2cd2efR99